packages/kbn-grid-layout/grid/types.ts

```diff
@@ -74,7 +77,7 @@ export interface PanelInteractionEvent {
   /**
    * The type of interaction being performed.
    */
-  type: 'drag' | 'resize' | 'drop';
+  type: 'drag' | 'resize';
```

Comment:
I was able to get an `onLayoutChange` callback to work without the `drop` event - so, I removed it